Core/CwCheat.cpp

```diff
@@ -87,7 +87,7 @@ bool CheatFileParser::Parse() {
 		getline(file_, line, '\n');
 		line = TrimString(line);
 
-		if (line.length() > 2 && line[0] == '_') {
+		if (line.length() >= 5 && line[0] == '_') {
```

Comment:
Hmm, there's a lot of checks of length inside `ParseLine()`.  If this is needed, it means one of them is buggy, right?

It might be good to add a comment indicating why 5, specifically, was chosen.  I suppose the shortest possible line would be `_G A` or something...

-[Unknown]

Reply:
The game name isn't really used by PPSSPP for anything, but yeah name is the one limiting the shortest part. Shortest name of PSP game has 2 letters(N+) and shortest name to display in UI would have 1 letter so since _C line has extra parameter they end up same at shortest of 5 letters.

 Guess I might add comment clearing that up, but that change wasn't really important here and doesn't do much, I made this change to keep it same with function called by cheat menu below as while it runs later it recreates list and would discard any invalid / too short lines that aren't comments anyway.